Client/core/Graphics/CRenderItemManager.cpp

```diff
@@ -312,8 +312,7 @@ CGuiFontItem* CRenderItemManager::CreateGuiFont(const SString& strFullFilePath,
 ////////////////////////////////////////////////////////////////
 void CRenderItemManager::NotifyContructRenderItem(CRenderItem* pItem)
 {
-    assert(!MapContains(m_CreatedItemList, pItem));
-    MapInsert(m_CreatedItemList, pItem);
+    assert(!m_CreatedItemList.insert(pItem).second); // assert if item was already in the collection.
```

Comment:
The standard library definition of assert might not actually compile the condition code: https://en.cppreference.com/w/cpp/error/assert. Although MTA uses it's own assert definition I still think it would be good practice to have required code outside of the assert. e.g.
```
bool bCheck = condition();
assert(bCheck);
```

Reply:
Okay, no problem.